uiowa/uiowa
# [AFR] File structure

Relates to https://github.com/uiowa/uiowa/issues/7966. 



# How to test

```
ddev blt ds --site=afr.fo.uiowa.edu && ddev drush @foafr.local config:import --source ../config/sites/afr.fo.uiowa.edu --partial -y &&  ddev drush @foafr.local cim -y &&  ddev drush @foafr.local uli /media/add/file
```

- Upload a file
- Confirm that file uploads to `sites/afr.fo.uiowa.edu/files/[filename]` directory
